desc/optimize/_constraint_wrappers.py

```diff
@@ -649,12 +659,10 @@ def build(self, use_jit=None, verbose=1):  # noqa: C901
             Level of output.
 
         """
-        eq = self._eq
         timer = Timer()
         timer.start("Proximal projection build")
 
-        self._eq = eq
-        self._linear_constraints = get_fixed_boundary_constraints(eq=eq)
+        self._linear_constraints = get_fixed_boundary_constraints(eq=self._eq)
```

Comment:
really these are `self._eq_linear_constraints`, might be clearer like that to separate from potential linear constraints that exist (outside of this Projection) on the actual bdry or other DOFs